Lib/test/test_long.py

```diff
@@ -1349,6 +1350,37 @@ def test_shift_bool(self):
                 self.assertEqual(type(value << shift), int)
                 self.assertEqual(type(value >> shift), int)
 
+    def test_as_integer_ratio(self):
+        tests = [10, 0, -10, 1]
+        for value in tests:
+            numerator, denominator = value.as_integer_ratio()
+            self.assertEqual((numerator, denominator), (value, 1))
+            self.assertIsInstance(numerator, int)
+            self.assertIsInstance(denominator, int)
+
+    def test_as_integer_ratio_maxint(self):
+        x = sys.maxsize + 1
+        self.assertEqual(x.as_integer_ratio()[0], x)
+
+    def test_as_integer_ratio_bool(self):
+        self.assertEqual(True.as_integer_ratio(), (1, 1))
+        self.assertEqual(False.as_integer_ratio(), (0, 1))
+        assert(type(True.as_integer_ratio()[0]) == int)
+        assert(type(False.as_integer_ratio()[0]) == int)
+
+    def test_as_integer_ratio_int_enum(self):
+        class Foo(enum.IntEnum):
+            X = 42
+        self.assertEqual(Foo.X.as_integer_ratio(), (42, 1))
+        assert(type(Foo.X.as_integer_ratio()[0] == int))
```

Comment:
This should be:  `self.assertEqual(type(Foo.X.as_integer_ratio()[0], int)`.  Unittest doesn't use assertion statements.